src/compat.rs

```diff
@@ -116,11 +116,11 @@ lazy_static! {
             if ABI::is_known(n) || n == 0 {
                 ABI::from(n)
             } else {
-                panic!("Unknown ABI: {}", n);
+                panic!("Unknown ABI: {n}");
```

Comment:
The error message for this should be fixed once you switch to the 2021 edition. In the 2015 and 2018 edition a panic with a single argument would use the given argument as a literal value to pass to `std::panic::panic_any`. With the 2021 edition it will use `format!()` for single argument `panic!()` too. (`std::panic::panic_any` was introduced to get the old behavior if necessary)

Reply:
Indeed, thanks!